de.metas.adempiere.adempiere/base/src/main/java/org/compiere/model/IQuery.java

```diff
@@ -233,16 +234,24 @@
 	IQuery<T> setOptions(Map<String, Object> options);
 
 	/**
-	 * Check if there items for query criteria
+	 * Check if there items for query criteria. Same functionality as {@link #anyExist()} ()}
 	 *
 	 * @return true if exists, false otherwise
 	 * @throws DBException
 	 */
 	boolean match() throws DBException;
 
 	/**
-	 * Returns an {@link Iterator} over current query selection.
+	 * Check if there items for query criteria. Same functionality as {@link #match()}
 	 *
+	 * @return true if exists, false otherwise
+	 * @throws DBException
+	 */
+	boolean anyExist() throws DBException;
```

Comment:
why introducing another method which is doing exaclty the same thing?

Reply:
because `match` doesn't tell me anything about what it would do, whereas `anyExist` says it right there in the name.

Reply:
In that case, propose a refactoring (=>i.e. make a PR)

Reply:
will make a new PR merging the `match` into `anyExist`

Reply:
why not `anyMatch()`?
... similar with java 8's  Stream.anyMatch.

Reply:
pls drop it. we aggreed to have this on a different PR